.travis.yml

```diff
@@ -120,15 +92,15 @@ matrix:
     - os: linux
       # Note: -fext-numeric-literals is needed to workaround gcc bug:
       # boost/math/constants/constants.hpp:269:3: error: unable to find numeric literal operator 'operatorQ'
-      env: BUILDTYPE=debug CXX="g++-6" CC="gcc-6" CXXFLAGS="-fext-numeric-literals"
+      env: BUILDTYPE=debug CXX="g++-9" CC="gcc-9" CXXFLAGS="-fext-numeric-literals"
```

Comment:
We install libstdc++10-dev and use gcc-9 and g++-9 here.
is there any reason to use different versions?

Reply:
`node-pre-gyp` requires python3.6+, when building with node16. By default travis uses `xenial` distribution for python, which has python3.5. So I had to use `focal` ubuntu distribution, which gives this error for `libstdc++-4.9-dev`
```
E: Unable to locate package libstdc++-4.9-dev
E: Couldn't find any package by glob 'libstdc++-4.9-dev'
E: Couldn't find any package by regex 'libstdc++-4.9-dev'
```

And for `g++-6`:
```
Package g++-6 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
```

Reply:
I mean if we install libstdc++ v10 why can't we use the same version for the gcc compilers instead of gcc-9?Or probably install libstdc++9-dev?

Reply:
I installed the `libstdc++9-dev` version and it builds fine. Thanks 👍